bwrap.xml

```diff
@@ -18,6 +18,11 @@
       <firstname>Colin</firstname>
       <surname>Walters</surname>
     </author>
+    <author>
+      <contrib>Developer</contrib>
+      <firstname>Simon</firstname>
+      <surname>McVittie</surname>
+    </author>
```

Comment:
I do not consider myself to be the primary maintainer of this project. I'm doing my best to keep it active and secure, but I have multiple other responsibilities, and since my best is clearly not good enough, I am not happy about labelling myself as having *even more* responsibility for this project.

Reply:
Dropped now.